src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+enum Color {
+    BLUE,
+    RED,
+    GREEN
+}
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
```

Comment:
The constructor parameter `color` should also be of the enum type `Color` instead of `String`. This will enforce that only valid colors defined in the enum can be used for a `Ball`.